docs/getting-started/tutorial-full-experience.asciidoc

```diff
@@ -0,0 +1,17 @@
+
+[[tutorial-your-data]]
+== Getting the full Kibana experience
+
+Ready for the full Kibana experience? This tutorial shows you how to:
+
+* Load a sample data set into Elasticsearch
+* Define an index pattern
+* Discover and explore the sample data
+* Visualize the data
+* Assemble visualizations into a dashboard
+* Inspect the data behind a visualization
+
+If security is enabled, you must have the `all` Kibana privilege to run this tutorial.
+You must also have the `create`, `manage` `read`, `write,` and `delete` 
+index privileges. See {xpack-ref}/security-privileges.html[Security Privileges] 
+for more information.
```

Comment:
I understand why this is necessary, but it seems a bit off putting to include in the second paragraph of my getting started experience. Is it possible to make it more subtle or push down in the page?

Reply:
Looks like this may have changed since but in the hosted docs you list the actual index pattern `.kibana_sample_data_flights`, this is actually `kibana_sample_data_flights` without the `.`